tools/nuget/ebpf-for-windows.nuspec.in

```diff
@@ -5,7 +5,7 @@
 <package>
 	<metadata>
 		<title>eBPF for Windows SDK</title>
-		<id>eBPF-for-Windows.{architecture}</id>
+		<id>eBPF-for-Windows.{architecture}{configuration}</id>
```

Comment:
This looks like it impacts the name of the nuget package. Is that acceptable for publishing in the public nuget feed?

@saxena-anurag @shankarseal @Alan-Jowett 

See https://github.com/microsoft/ebpf-for-windows/blob/ae8771db19788e47c5600f1b009f81112d114ca2/docs/ReleaseProcess.md#L4 which references this in some manual steps.

Reply:
The name for the release configuration stays the same. The change in  `set-version.ps1` change now adds the configuration for non-release configurations.

Example:
`eBPF-for-Windows.x64.x.y.z` for the "NativeOnlyRelease" configuration (no change)
`eBPF-for-Windows.x64.NativeOnlyDebug.x.y.z` for the "NativeOnlyDebug" configuration (new name)

This allows both of them to be published for a given version.